blog/_posts/2016-09-21-Buffering.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Buffering
+layout: post
+created_at: 2016-09-21
+permalink: blog/2016-09-19-Buffering
+current: blog
+author: Thea Amanda Kupler and Marie-Luise Kochsiek
+twitter: @loadtocode
```

Comment:
please remove the `@` symbol, as it breaks Jekyll and keeps it from building properly.